packages/searchbase/src/schema/elasticsearch.js

```diff
@@ -0,0 +1,251 @@
+import { componentsAlias, dataTypes } from '../utils';
+
+const { SEARCHBASE, SEARCHBOX, SEARCHCOMPONENT } = componentsAlias;
+
+export default {
+  url: {
+    components: [SEARCHBASE],
+    type: dataTypes.STRING,
+    required: true
+  },
+  index: {
+    components: [SEARCHBASE, SEARCHBOX, SEARCHCOMPONENT],
```

Comment:
@mohdashraf010897 Why do you have `searchbox` here? Schema is to only validate the `searchbase`(headless) components.